lib/mixins/property-accessors.html

```diff
@@ -563,6 +568,8 @@
 
     }
 
+    return PropertyAccessors;
```

Comment:
Investigate if separating return from class definition is necessary; not used above.